scss/6-components/_navigation.scss

```diff
@@ -6,7 +6,7 @@
 .cads-navigation {
   display: flex;
   justify-content: space-between;
-  background-color: $cads-language__brand-primary;
```

Comment:
This was using too broad a variable, so needed to make a dedicated language var for this to avoid changing too much in the docs site context